src/nhp/aci/config.py

```diff
@@ -8,21 +8,20 @@
 from dotenv import load_dotenv
 
 
-def load_env_files() -> None:
+def load_env_files(config_dir: str) -> None:
```

Comment:
can we please use the suggestion I made? a) it used a standard library in python for handling the loading from config folders and b) was pure, so easier to test, and requires very little changes to the existing test for create_from_envvars.

Reply:
correction, platformdirs is not part of the standard library... not sure why I assumed that. but it is included now by a bunch of other libraries. so we should add that as a dependency.

but, it does correctly handle the paths for config on Windows, Linux, and Mac.

There is a subtlty that Windows could have different places that config could live in, depending on how the OS is configured afaik, this package does handle some of those cases